ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -4,10 +4,55 @@
 ~}}
 
 <SyncHeader
-  @title="Create destination"
+  @title="Create destination for {{this.displayName}} "
```

Comment:
Consulted design and they want to add the destination name here i.e. "AWS Secrets Manager"